mods/cnc/maps/gdi02/gdi02.lua

```diff
@@ -80,7 +80,7 @@ WorldLoaded = function()
 
 	nodObjective = enemy.AddPrimaryObjective("Destroy all GDI troops.")
 	gdiObjective1 = player.AddPrimaryObjective("Eliminate all Nod forces in the area.")
-	gdiObjective2 = player.AddSecondaryObjective("Capture the Tiberium Refinery.")
+	gdiObjective2 = player.AddSecondaryObjective("Capture the Tiberium refinery.")
```

Comment:
http://cnc.wikia.com/wiki/Tiberium_refinery has "refinery" in lower case - also, other types of refineries would not be capitalized (oil refinery, etc.). I suspect that Refinery is uppercase there since it's a heading.
I considered making tiberium lowercase too (it's like an element, and elements like sodium are not written with capitals either) but seeing it capitalized everywhere else I kept it that way